.ci/install_cri_containerd.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+cri_repository="github.com/containerd/cri"
+
+die() {
+	echo >&2 -e "ERROR: $*"
+	exit 1
+}
+
+info() {
+	echo -e "INFO: $*"
+}
+
+source "${script_dir}/lib.sh"
+cri_containerd_version=$(get_version "externals.cri-containerd.version")
+containerd_version=$(get_version "externals.cri-containerd.meta.containerd-version")
+
+
+case "$(arch)" in
+	"aarch64")
+		ARCH=arm64
+		;;
+
+	"x86_64")
+		ARCH=amd64
+		;;
+	"*")
+		die "Arch $(arch) not supported"
+		;;
+esac
+
+
+go get github.com/containerd/cri
+pushd "${GOPATH}/src/${cri_repository}" >> /dev/null
```

Comment:
`clone_and_build` call here?

Reply:
it still need to do some extra steps in the repository  - that is why I just do a go get